docs/sources/alerting/configure-notifications/create-notification-policy.md

```diff
@@ -55,38 +55,33 @@ For more information on notification policies, refer to [fundamentals of Notific
    - **Repeat interval** Minimum time interval for re-sending a notification if no new alerts were added to the group. Default is 4 hours.
 1. Click **Save** to save your changes.
 
-## Add new nested policy
+## Add a child policy
 
-To create a new notification policy, you need to follow its tree structure. New policies created on the trunk of the tree (default policy), are the tree branches. And, subsequently, each branch can bear their own child policies. This is why you always add a new **nested** policy under either the default policy, or under a already nested policy.
-
-1. In the left-side menu, click **Alerts & IRM** and then **Alerting**.
-2. Click **Notification policies**.
-3. From the **Choose Alertmanager** dropdown, select an Alertmanager. By default, the **Grafana Alertmanager** is selected.
-4. To add a top level specific policy, go to the Specific routing section (either to the default policy, or to another existing policy in which you would like to add a new nested policy) and click **+New nested policy**.
-5. In the Matching labels section, add one or more rules for matching alert labels.
-6. In the **Contact point** dropdown, select the contact point to send notification to if alert matches only this specific policy and not any of the nested policies.
-7. Optionally, enable **Continue matching subsequent sibling nodes** to continue matching sibling policies even after the alert matched the current policy. When this option is enabled, you can get more than one notification for one alert.
-8. Optionally, enable **Override grouping** to specify the same grouping as the default policy. If this option is not enabled, the default policy grouping is used.
-9. Optionally, enable **Override general timings** to override the timing options configured in the group notification policy.
-10. Click **Save policy** to save your changes.
-
-## Add nested policy
+Child policies are always added under the default policy and are automatically added to the end of the tree structure. If you want to choose where to position your policy, see the section on **Add a sibling policy**.
 
 1. In the left-side menu, click **Alerts & IRM** and then **Alerting**.
 1. Click **Notification policies**.
-1. Expand the specific policy you want to update.
-1. Click **+ Add nested policy**, then add the details using information in [Add new specific policy](#add-new-nested-policy).
+1. From the **Choose Alertmanager** dropdown, select an Alertmanager. By default, the **Grafana Alertmanager** is selected.
+1. Click **+New child policy** from the default policy.
```

Comment:
You can add a child policy to _any_ policy not just the default policy. It's just the only action you can take on the default policy.